resources/spotlight.rb

```diff
@@ -32,8 +32,15 @@ def desired_spotlight_state
 end
 
 action :set do
+  volume = MetadataUtil.new(target_volume)
+
+  execute 'Enable Spotlight server' do
+    command volume.toggle_spotlight_server(true)
```

Comment:
I'm pretty sure we can use the `macosx_service` resource here. See my other comment.